Makefile

```diff
@@ -36,7 +36,7 @@ all:	build man
 new:	clean all
 
 version:
-	sed -i "" -e "/^VERSION :=/s/:= .*/:= ${RVERSION}/" python/Makefile src/Makefile
```

Comment:
I take it back - removing the space works for linux, but it does NOT work as I expected it to in OSX.

I'm thinking (and it's a kludge), that I could add `Makefile-e` to the `.gitignore` file and `make clean` target.